README.md

```diff
@@ -1 +1,265 @@
+* [Sample Alerts](#sample-alerts)
+* [Find Alerts](#find-alerts)
+
+# Filtering
+## Post a Filter
+Adds a filter to the database.
+
+**Endpoint**: `POST "/filter"`\
+**Body**:
+```
+{
+    "pipeline": aggregate pipeline (array of bson documents),
+    "catalog": catalog name (string),
+    "permissions": allowed permissions,
+    "id": filter id (i32)
+}
+```
+
+**Example Body**:
+```
+{
+    "pipeline": 
+    [   
+        {
+            "$project": {
+                "cutoutScience": 0, 
+                "cutoutDifference": 0, 
+                "cutoutTemplate": 0, 
+                "publisher": 0, 
+                "schemavsn": 0
+            }
+        }, 
+        {
+            "$lookup": {
+                "from": "alerts_aux", 
+                "localField": "objectId", 
+                "foreignField": "_id", 
+                "as": "aux"
+            }
+        }, 
+        {
+            "$project": {
+                "objectId": 1, 
+                "candid": 1, 
+                "candidate": 1, 
+                "classifications": 1, 
+                "coordinates": 1, 
+                "prv_candidates": {
+                    "$arrayElemAt": [
+                        "$aux.prv_candidates", 
+                        0
+                    ]
+                }, 
+                "cross_matches": {
+                    "$arrayElemAt": [
+                        "$aux.cross_matches", 
+                        0
+                    ]
+                }
+            }
+        }, 
+        {
+            "$match": {
+                "candidate.drb": {
+                    "$gt": 0.5
+                }, 
+                "candidate.ndethist": {
+                    "$gt": 1.0
+                }, 
+                "candidate.magpsf": {
+                    "$lte": 18.5
+                }
+            }
+        }
+    ],
+    "catalog": "ZTF",
+    "permissions": [1],
+    "id": -3
+}
+```
+
+## Add a New Filter Version
+Adds a new pipeline to a filter's pipeline array and sets the filter's active pipeline id to the new pipeline's id.
+
+**Endpoint**: `PATCH "/filter/{filter_id}"`\
+**Body**:
+```
+{
+    "pipeline": aggregate pipeline (array of bson documents)
+}
+```
+
+**Example Body**:
+```
+{
+    "pipeline": [
+        {
+            "$project": {
+                "cutoutScience": 0, 
+                "cutoutDifference": 0, 
+                "cutoutTemplate": 0, 
+                "publisher": 0, 
+                "schemavsn": 0
+            }
+        },
+        {
+            "$match": {
+                "candidate.drb": {
+                    "$gt": 0.5
+                }, 
+                "candidate.ndethist": {
+                    "$gt": 1.0
+                }, 
+                "candidate.magpsf": {
+                    "$lte": 18.5
+                }
+            }
+        }
+    ]
+}
+```
+
+# Querying
+
+## Get Object
+Retreives the most recent detection of an object from the specified catalog.
```

Comment:
small typo here at `retrieves`. Also I mean we don't just retrieve the most recent detection, we get lightcurves and other dataproducts. Perhaps better to say here that it retrieves an object with its lightcurve, crossmatches, metadata, and images.